src/views/SoraCard.vue

```diff
@@ -190,8 +191,9 @@ export default class SoraCard extends Mixins(mixins.LoadingMixin, SubscriptionsM
     await this.unsubscribeFromTotalXorBalance();
   }
 
-  mounted(): void {
+  async mounted(): Promise<void> {
     this.checkKyc();
+    await this.getFees();
```

Comment:
do you really need async await here?
- this is the last line in this method
- the life cycle won't wait for it